indexstyle.css

```diff
@@ -117,6 +119,8 @@ li a:hover {
 .intropage .container {
 	height: 100vh;
 	align-items: center;
+	background-image: url(handshake.jpg);
```

Comment:
Consider changing this background in your media queries, on a phone it looks a bit strange.

Reply:
agreed. done